### ISSUE TITLE
Issues with notifications

### ISSUE THREAD

**Issue by hir1904** [Icinga/icingaweb2-module-director#1041]
Current Behavior
Cant deploy my configuration with a notification set-up, following error occurs 

```information/cli: Icinga application loader (version: r2.6.3-1)
information/cli: Loading configuration file(s).
information/ConfigItem: Committing config item(s).
information/ApiListener: My API identity: ***
critical/config: Error: Validation failed for object 'mysite.de!Email-Alert-Hosts' of type 'Notification'; Attribute 'command': Object 'mail-host-notification' of type 'NotificationCommand' does not exist.
Location: in [stage]/zones.d/master-zone/notification_templates.conf: 2:5-2:38
[stage]/zones.d/master-zone/notification_templates.conf(1): template Notification "Email-Alert-Host" {
[stage]/zones.d/master-zone/notification_templates.conf(2):     command = "mail-host-notification"
                                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[stage]/zones.d/master-zone/notification_templates.conf(3):     states = [ Critical, Down, Up, Warning ]
[stage]/zones.d/master-zone/notification_templates.conf(4):     types = [ Problem, Recovery ]

critical/config: 1 error
```
Although the command exists, the kickstart wizard imported it as an external command "mail-host-notification | /etc/icinga2/scripts/mail-host-notification.sh"

## Your Environment

* Director version (System - About): latest as of 27.07.2017 313c816
* Icinga Web 2 version and modules (System - About): 2.4.1
* Icinga 2 version (`icinga2 --version`): (version: r2.6.3-1)
* Operating System and version: debian 8 jessie
* Webserver, PHP versions: apache2, php5

**Comment by Thomas-Gelf** [comment]
Well... mail-host-notification usually exists in your local sample configuration and is imported by the Director at kickstart time. In case you removed the include line for `conf.d` from your `icinga2.conf` (something I always do, as I do not want a sample config from upstream packages to mess with my monitored objects), you also removed the reference to the definition of that command. Either copy the content of `conf.d/commands.conf` or include just that single file from that directory and you should be covered.

Regards,
Thomas

**Comment by hir1904** [comment]
Hey @Thomas-Gelf,

thanks for the quick response! As in one of the recent webinars (which i totally like!) I removed everything in the conf.d folder except for the api-users.conf file. If i exclude the conf.d now, nothing really changes except I dont have any API-Users defined anymore. I copied the commands.conf file over to the conf.d folder and I can deploy the config now. So I guess the problem can be closed for now.

Best Regards
